### ISSUE TITLE
Tags describing the project

### ISSUE THREAD

**Issue by pieceofeden** [thealphadollar/brkout#38]
some tags describing the project in README.md to help readers and also the link to the communication channel

**Comment by thealphadollar** [comment]
Yeah, you can add tags such as "made with python", "link to gitter channel" etc.

**Comment by pieceofeden** [comment]
Okay !!
